src/components/stillage/StillageList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react'
+import { styled } from '@linaria/react'
+import StillageItem from './StillageItem'
+import { useDispatch, useSelector } from 'react-redux'
+import { RootState } from '../../redux/store'
+import { clearStillageList, setStillageList } from '../../redux/slices/stillageSlice'
```

Comment:
I would use absolute path instead of relative path, e.g.

```javascript
import { clearStillageList, setStillageList } from '@redux/slices/stillageSlice'
```

It resolves 2 problems:
1. You may not worry about path length and how much `../` you need to add to get a module.
2. When you will refactor this module or others, or whole file-system of your app, you will handle import change easy.

To set up relative paths, you need to read about them for your bundler. As you use CRA, you need to read about relative paths for [webpack](https://webpack.js.org/configuration/resolve/) and CRA. But I strongly recommend you to use [vite](https://vitejs.dev/) as bundler and [its react plugin](https://vitejs.dev/guide/#scaffolding-your-first-vite-project). CRA is meant to be deprecated.

Relative paths is a popular problem, so you can easily find a lot of articles about it, like this
[Escaping the relative import path hell, in Javascript](https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e)

Reply:
I've changed relative paths to absolute ones